src/core/RoutesFolderWatcher.ts

```diff
@@ -41,7 +42,7 @@ export class RoutesFolderWatcher {
         return
       }
       handler({
-        filePath,
+        filePath: normalize(filePath),
```

Comment:
let's move `filePath = normalize(filePath)` up so it's passed to everything

Reply:
@posva OK, I'll deal with the problem after work。